qt_ui/windows/QLiberationWindow.py

```diff
@@ -266,6 +269,16 @@ def saveGameAs(self):
             liberation_install.setup_last_save_file(self.game.savepath)
             liberation_install.save_config()
 
+            self.updateWindowTitle(file)
+
+    def updateWindowTitle(self, file):
```

Comment:
New code needs types, and the name file should be avoided because it shadows the builtin type.

```
def update_window_title(self, save_path: Optional[str]) -> None:
```

(made optional to accommodate the case where the game failed to load, or for first launch)